conda_smithy/templates/README.md.tmpl

```diff
@@ -14,24 +14,31 @@ Summary: {{ package.meta.about.summary }}
 Installing {{ package.name() }}
 ==========={{ '=' * package.name()|length }}
 
-Installing {{ package.name() }} from the conda-forge channel can be achieved by adding conda-forge to your channels with:
+Installing `{{ package.name() }}` from the `{{ channels.targets[0][0] }}` channel can be achieved by adding `{{ channels.targets[0][0] }}` to your channels with:
```

Comment:
How about capturing ```channels.targets[0][0]``` into a ``channel_name`` variable at the beginning of this document and using that?